utk/sass/_custom.scss

```diff
@@ -411,6 +411,84 @@ a.thumbnail:focus {
     font-weight: 500;
 }
 
+//
+//Flags
+//---------------------------------------------------
+.flag{
```

Comment:
I'm not against this style, but a more compact way to do this is to use a SCSS style constant and just pulling it into the 'flag-STYLE' classes. This way you only need a 'flag-STYLE' class in the HTML.

But yeah, this is fine.

Reply:
ok, so do you mean not doing the & in front of the flag-* ?